tests/integration/sdk_api/projects/test_project.py

```diff
@@ -12,16 +12,42 @@
 class TestProject(tests.integration.sdk_api.base.TestMLRunIntegration):
     def test_create_project(self):
         project_name = "some-project"
-        project = mlrun.new_project(project_name)
-        project.save_to_db()
+        mlrun.new_project(project_name)
         projects = mlrun.get_run_db().list_projects()
         assert len(projects) == 1
         assert projects[0].metadata.name == project_name
 
+    def test_create_project_failure_already_exists(self):
+        project_name = "some-project"
+        mlrun.new_project(project_name)
+
+        with pytest.raises(mlrun.errors.MLRunConflictError) as exc:
+            mlrun.new_project(project_name)
+        assert (
+            f"Project with name {project_name} already exists. Use override=True to override the existing project."
+            in str(exc.value)
+        )
+
+    def test_override_project(self):
+        project_name = "some-project"
+
+        # verify override does not fail if project does not exist
+        mlrun.new_project(project_name, override=True)
+        projects = mlrun.get_run_db().list_projects()
+        assert len(projects) == 1
+        assert projects[0].metadata.name == project_name
+        old_creation_time = projects[0].metadata.created
+
+        mlrun.new_project(project_name, override=True)
```

Comment:
Need to also create some objects in the project (function, artifacts, whatever) - this should verify the deletion of a non-empty project.